packages/field-base/test/shadow-focus-mixin.test.js

```diff
@@ -202,7 +204,28 @@ describe('shadow-focus-mixin', () => {
       expect(customElement.hasAttribute('focused')).to.be.true;
     });
 
-    it('should not focus if the focus is not received from outside', () => {
+    it('should delegate focus if the focus is received from outside using keyboard navigation', async () => {
+      const spy = sinon.spy(focusElement, 'focus');
+      await sendKeys({ press: 'Tab' });
+      expect(spy.called).to.be.true;
+    });
+
+    it('should not delegate focus when clicking on non-focusable child', async () => {
+      const span = document.createElement('span');
+      span.textContent = 'test';
+      customElement.appendChild(span);
+
+      const spy = sinon.spy(focusElement, 'focus');
+      // Click center of span child element
+      const rect = span.getBoundingClientRect();
+      const middleX = Math.floor(rect.x + rect.width / 2);
+      const middleY = Math.floor(rect.y + rect.height / 2);
+      await sendMouse({ type: 'click', position: [middleX, middleY] });
```

Comment:
It is recommended to use the `resetMouse` command after each test to reset the mouse position.

Reply:
Done